### PR TITLE
Maintenance Release v1.2.0

### PR DESCRIPTION
**Pull request by scottqueen-bixal**
## PR Summary

Maintenance Release v1.2.0

## Test Environment

https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.sites.pages.cloud.gov/preview/gsa/usagov-benefits-eligibility/main/

## Release Draft

https://github.com/GSA/usagov-benefits-eligibility/releases

## Related Github Issues

https://github.com/orgs/GSA/projects/48/views/17?filterQuery=release%3A%221.2.0%22

## Type of change

- [x] Task
- [x] Bug fix
- [x] Content
- [ ] New feature
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by FatmaBakir — APPROVED** [review]
Approved!

File: package.json
Comment (FatmaBakir) [inline package.json:1]: The `version` needs to be bumped to reflect the release tag.

Comment (FatmaBakir) [inline package.json:1]: The version needs to be bumped to reflect the release tag.

**Review by FatmaBakir — CHANGES_REQUESTED** [review]

File: package.json
Comment (scottqueen-bixal) [reply package.json:1]: updated

**Review by FatmaBakir — APPROVED** [review]

**Review by nehemiah-abuga — APPROVED** [review]

**Pull request merged**